### PR TITLE
Add spanish localization

### PR DESCRIPTION
**Pull request by DestroyerIV**

### CODE REVIEW

File: src/localizations/es.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+  emptyWorkspace_title: () => `Comienza a trabajar en tu tema`,
+  emptyWorkspace_createTheme: () => `Crear nuevo tema`,
+  emptyWorkspace_openTheme: () => `Abrir temas existentes`,
+  emptyWorkspace_credits: () => <React.Fragment>
+    Creador por {}
+    <Link href="//t.me/snejugal">@snejugal</Link>
+    {} y {}
+    <Link href="//t.me/AlexStrNik">@AlexStrNik</Link>
+    {} con {}
```
Comment (snejugal) [inline src/localizations/es.js:28]: Add yourself here, something like `Created by ... and translated by <Link href="//t.me/NeganLucille">@NeganLucille</Link> with <LoveEmoji/>`

File: src/localizations/es.js [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+  variableEditor_colorModelsTab: () => `Modelos de colores`,
+  variableEditor_palettesTab: () => `Paletas`,
+  variableEditor_wallpaperColorsHint: () => `Aquí hay algunos colores de fondo de pantalla. Haz clic en uno para agregarlo a la paleta de tu tema:`,
+  variableEditor_editPalette: () => `Edita la paleta`,
+  variableEditor_themeColorsPlaceholder: () => `Vaya, todavía no hay colores en el tema.`,
+  variableEditor_themeCustomPalettePlaceholder: () => `Vaya, todavía no hay colores en la paleta personalizada. Haz clic en el boton "Editar la paleta."`,
+
+  scriptRunner_title: () => `Ejecuta un script`,
+  scriptRunner_description: () => <React.Fragment>
+    .attheme editor le permite ejecutar facilemente scripts escritos en EcmaScript 2017 para acelerar el desarrollo de temas. Puede leer mas sobre la API que proporciona el editor <Link href="//github.com/SnejUgal/attheme-editor/wiki/.attheme-editor-scripts-documentation"> en la wiki de su repositorio de Github </Link>.
```
Comment (SnejUgal) [inline src/localizations/es.js]: Remove the space at the end

File: src/localizations/es.js [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+  workspace_unusedVariable: () => `No utilizado por Telegram`,
+  workspace_obsoleteVariable: () => `Obsoleta`,
+  workspace_nonStandardVariable: () => `No Estándar`,
+  workspace_search: () => `Buscar`,
+  workspace_variablesAmount: ({ total, theme }) => {
+    const forms = {
+      "one one": `${theme} of ${total} variable se agrega al tema`,
+      "one other": `${theme} of ${total} variables se agregan al tema`,
+      "other one": `${theme} of ${total} variable se agrega al tema`,
+      "other other": `${theme} of ${total} las variables se agregan al tema`,
```
Comment (snejugal) [inline src/localizations/es.js:66]: I'm not a Spanish speaker, but I think these should be reviewed (I guess `se agrega` is linked to `n of m variables`. Here are some examples of forms when applied:
`1 of 1 variable is added to the theme`
`1 of 2 variables is added to the theme`
`2 of 1 variables are added to the theme`
`2 of 2 variables are added to the theme`

**Review by snejugal — CHANGES_REQUESTED** [review]
see comments

**Comment by snejugal** [comment]
Whops, forgot to close this PR once I merged #26

**Pull request closed without merging**